font.css

```diff
@@ -0,0 +1,8 @@
+@font-face {
+    font-family: 'Roboto';
+    src: url('font/subset-Roboto-Bold.woff2') format('woff2'),
+        url('font/subset-Roboto-Bold.woff') format('woff');
+    font-weight: bold;
+    font-style: normal;
+    font-display: swap;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/evagame/pullRequest?prid=5788637)